lib/md040.js

```diff
@@ -2,16 +2,30 @@
 
 "use strict";
 
-const { addErrorContext, filterTokens } = require("../helpers");
+const { addError, addErrorContext, filterTokens } = require("../helpers");
 
 module.exports = {
   "names": [ "MD040", "fenced-code-language" ],
   "description": "Fenced code blocks should have a language specified",
   "tags": [ "code", "language" ],
   "function": function MD040(params, onError) {
+    let allowed = params.config.allowed_languages;
+    allowed = Array.isArray(allowed) ?
+      allowed.map((lang) => lang.toLowerCase()) :
+      null;
```

Comment:
Like Nick suggested, using [] instead of null avoids the need to compare against null on line 22.